cluster-autoscaler/cloudprovider/gce/gce_url.go

```diff
@@ -35,20 +35,41 @@ func ParseMigUrl(url string) (project string, zone string, name string, err erro
 	return parseGceUrl(url, "instanceGroups")
 }
 
+// ParseIgmUrl expects url in format:
+// https://content.googleapis.com/compute/v1/projects/<project-id>/zones/<zone>/instanceGroupManagers/<name>
+func ParseIgmUrl(url string) (project string, zone string, name string, err error) {
```

Comment:
Do we call this anywhere?

Reply:
Now we do (3 times in `gce_manager.go`). Thanks for catching this!